src/TensorFlowNET.Core/Eager/c_api.eager.cs

```diff
@@ -114,13 +114,13 @@ public partial class c_api
         /// <param name="status">TF_Status*</param>
         /// <returns>TFE_Context*</returns>
         [DllImport(TensorFlowLibName)]
-        public static extern TFE_Context TFE_NewContext(IntPtr opts, SafeStatusHandle status);
+        public static extern SafeContextHandle TFE_NewContext(IntPtr opts, SafeStatusHandle status);
 
         [DllImport(TensorFlowLibName)]
-        public static extern TFE_Context TFE_ContextStartStep(IntPtr ctx);
+        public static extern void TFE_ContextStartStep(SafeContextHandle ctx);
 
         [DllImport(TensorFlowLibName)]
-        public static extern TFE_Context TFE_ContextEndStep(IntPtr ctx);
+        public static extern void TFE_ContextEndStep(SafeContextHandle ctx);
```

Comment:
⚠️ I changed the signature of this P/Invoke definition. I believe the form here is correct but it should be verified.